core/src/main/java/io/hyperfoil/core/generators/ObjectValueProviderBuilder.java

```diff
@@ -0,0 +1,74 @@
+package io.hyperfoil.core.generators;
+
+import io.hyperfoil.api.config.BenchmarkDefinitionException;
+import io.hyperfoil.api.config.InitFromParam;
+import io.hyperfoil.api.session.Access;
+import io.hyperfoil.api.session.Session;
+import io.hyperfoil.core.session.SessionFactory;
+import io.hyperfoil.function.SerializableToObjectFunction;
+
+public abstract class ObjectValueProviderBuilder<P, V> implements InitFromParam<ObjectValueProviderBuilder<P, V>> {
```

Comment:
Ints are not Objects, so I'd rather call it `BaseValueProviderBuilder`.

Reply:
Renamed to `BaseValueProviderBuilder `